petab/parameters.py

```diff
@@ -26,22 +27,49 @@ def get_parameter_df(
     if parameter_file is None:
         return parameter_file
 
+    parameter_df = None
+
+    if isinstance(parameter_file, pd.DataFrame):
+        parameter_df = parameter_file
+
     if isinstance(parameter_file, str):
-        parameter_file = pd.read_csv(parameter_file, sep='\t')
+        parameter_df = pd.read_csv(parameter_file, sep='\t')
+
+    if isinstance(parameter_file, list):
+        for parameter_subset_file in parameter_file:
```

Comment:
would rather
`parameter_df = pd.concat([pd.read_csv(file, sep='\t') for file in parameter_file])`
and then just perform duplicate checks once

Reply:
Removal of the `for` loop also meant removal of the file, which contained a copy of the duplicate, from the error message (such reporting may be more suited for `petablint`, where all files that contain copies of duplicates could be reported).

Reply:
yeah I think thats fine, there is anyways a ~50% chance that the reported file is not the one you actually want to remove the parameter from ;)